torchci/components/TestInsights.tsx

```diff
@@ -0,0 +1,60 @@
+import { JobData } from "../lib/types";
+import React from "react";
+
+const NOT_SUPPORTED_JOBS = [
```

Comment:
Added. They are not supported at the moment because neither the monitoring script is running there at the moment (libtorch, bazel, android) nor the CI job uploads to S3 due to connectivity issues (macos, rocm).  I have plan to fix macos and rocm soon, so they will be supported